crypto/fipsmodule/ec/ec_key.c

```diff
@@ -325,6 +326,42 @@ int EC_KEY_check_key(const EC_KEY *eckey) {
   return 1;
 }
 
+static int EVP_EC_KEY_check_fips(EC_KEY *key) {
+  uint8_t msg[16] = {0};
+  size_t msg_len = 16;
+  int ret = 0;
+  uint8_t* sig_der = NULL;
+  EVP_PKEY* evp_pkey = EVP_PKEY_new();
+  EVP_MD_CTX* ctx = EVP_MD_CTX_new();
+  const EVP_MD* hash = EVP_sha256();
```

Comment:
Just continuing the nitty comment: I think in other places below and above in this file and elsewhere the `*` is next to the declared variable and not to the type.
```suggestion
  EVP_PKEY *evp_pkey = EVP_PKEY_new();
  EVP_MD_CTX *ctx = EVP_MD_CTX_new();
  const EVP_MD *hash = EVP_sha256();
```